src/config.js

```diff
@@ -0,0 +1,3 @@
+
+// This is where requests for assets included by mfr.js should be made
+__webpack_public_path__ = "http://localhost:7778/assets/";
```

Comment:
This is hardcoded now, but could be set up with an ENV variable for pointing things at a cdn etc